org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/configurator/ProjectConfigurationRequest.java

```diff
@@ -23,19 +23,22 @@
 import java.util.Map;
 import org.eclipse.core.resources.IFile;
 import org.eclipse.core.resources.IProject;
+import org.sonarlint.eclipse.core.analysis.IAnalysisConfigurator;
 
+/**
+ * @deprecated since 2.7 use {@link IAnalysisConfigurator}
+ */
+@Deprecated
 public class ProjectConfigurationRequest {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)